package.json

```diff
@@ -68,7 +67,7 @@
     "@rollup/rollup-linux-x64-gnu": "*"
   },
   "peerDependencies": {
-    "@vitest/browser": "2.1.0",
+    "@vitest/browser": "^2.1.0",
     "webdriverio": "^8.39.0"
```

Comment:
If the `install-browser-deps` now installs specific versions of these dependencies, should we then also remove the caret here to ensure this specific version is installed?

Reply:
I tried this but when I run the `npm run install-browser-deps` script, it automatically adds the caret back (no idea why).  It should be okay though since we're installing specific versions from the script.